backend/app/utils/ai_processors.py

```
@@ -331,3 +269,118 @@
+        
+        # 更新失败状态
+        content_item.processing_status = "failed"
+        content_item.error_message = str(e)
+        content_item.last_processed_at = datetime.utcnow()
+        session.add(content_item)
+        session.commit()
+        
+        return {}
+
+def has_recent_ai_analysis(
+    content_item: ContentItem, 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)